scipy/io/matlab/mio5_utils.pyx

```diff
@@ -914,17 +914,16 @@ cdef class VarReader5:
             int *n_duplicates
             char **name_ptrs
```

Comment:
You still need to remove this and the call to `free(name_ptrs)` on line 935.

Reply:
Done. Sorry I missed that earlier.